catalog-spec/catalog-spec.md

```diff
@@ -85,13 +85,14 @@ For a full discussion of the situations where relative and absolute links are re
-| root   | STRONGLY RECOMMENDED. URL to the root STAC Catalog or [Collection](../collection-spec/README.md). Catalogs should include a link to their root, even if it's the root and points to itself.                                                                     |
-| parent | URL to the parent STAC entity (Catalog or Collection). Non-root Catalogs should include a link to their parent.                                                                                                                                                 |
-| child  | URL to a child STAC entity (Catalog or Collection).                                                                                                                                                                                                             |
-| item   | URL to a STAC Item.                                                                                                                                                                                                                                             |
+| Type       | Description                                                                                                                                                                                                                                                                      |
+| ---------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| self       | STRONGLY RECOMMENDED. *Absolute* URL to the location that the Catalog file can be found online, if available. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from.                  |
+| root       | STRONGLY RECOMMENDED. URL to the root STAC Catalog or [Collection](../collection-spec/README.md). Catalogs should include a link to their root, even if it's the root and points to itself.                                                                                      |
+| parent     | URL to the parent STAC entity (Catalog or Collection). Non-root Catalogs should include a link to their parent.                                                                                                                                                                  |
+| collection | URL to a parent Collection. *Absolute* URLs should be used whenever possible. The referenced Collection is STRONGLY RECOMMENDED to implement the same STAC version as the Catalog. A link with this `rel` type is *required* if the `collection` field in properties is present. |
```

Comment:
I think the changelog should mention that we add the collection relation type for catalogs.